java/mtsj/core/src/main/resources/db/migration/V0005__R001_Master_data.sql

```diff
@@ -1,9 +1,9 @@
 INSERT INTO UserRole(id, modificationCounter, name, active) VALUES (0, 1, 'Customer', true);
 INSERT INTO UserRole(id, modificationCounter, name, active) VALUES (1, 1, 'Waiter', true);
 INSERT INTO UserRole(id, modificationCounter, name, active) VALUES (2, 1, 'Manager', true);
-INSERT INTO User(id, modificationCounter, username, password, email, idRole) VALUES (0, 1, 'user0', 'password', 'user0@mail.com', 0);
-INSERT INTO User(id, modificationCounter, username, password, email, idRole) VALUES (1, 1, 'waiter', 'waiter', 'waiter@mail.com', 1);
-INSERT INTO User(id, modificationCounter, username, password, email, idRole) VALUES (2, 1, 'manager', 'manager', 'manager@mail.com', 2);
+INSERT INTO User(id, modificationCounter, username, password, email, idRole) VALUES (0, 1, 'user0', '{bcrypt}$2y$12$QkELlzmG9R0ITyR18FhbFuHa./9xnInTMJSl7OEKuROB1m4AFPMhq', 'user0@mail.com', 0);
```

Comment:
nice, that you even improved this. Still we need to clarify the strategy as there are also redundant passwords in `WebSecurityConfigurer`. But great that you solved this issue systematically.